README.md

```diff
@@ -2,16 +2,95 @@
 
 ## Roo Packaging and Installation
 
-### Updating Version (for a new release)
+### Setup
 
-1. Bump the version in `package.json`
-2. Update the version number in the `files` list in `package.json`
+- Install dependencies:
+   ```bash
+   npm run install:all
+   ```
+- Now you can either:
+  1. Install the latest extension from `bin/roo-cline-<latest_version>.vsix`, skip the packaging steps below
+  2. Or build the extension from source and proceed to the packaging steps below
+
+
+### Packaging
+1. **If** you have new changes, bump the version in `package.json`
+    - Remove the old VSIX file:
+    ```bash
+    rm bin/roo-cline-*.vsix
+    ```
+2. Build the VSIX file:
+   ```bash
+   npm run build
+   ```
+3. The new VSIX file will be created in the `bin/` directory
+4. Commit the new VSIX file to git:
+   ```bash
+   git add bin/*.vsix
+   git commit -m "chore: update VSIX to version <new_version>"
+   ```
+
+### Installation
+- **Option 1:** Follow these [instructions](https://www.cursor.com/how-to-install-extension) to manually drag the `.vsix` file into Cursor's Extensions panel (Cmd/Ctrl+Shift+X) and install it.
+
+- **Option 2:** Make sure you have the Cursor CLI installed and in your PATH.
```

Comment:
I think we might just want to get rid of these instructions for manual installation, but if not, I think we should make them more generic for all vscode compatible editors.

Reply:
will make them more generic, I still manually install when testing builds and cline changes